pkg/multicluster/register/helpers.go

```diff
@@ -168,7 +192,14 @@ func RegisterProviderClusterFromConfig(
 	opts RbacOptions,
 	registrant ClusterRegistrant,
 	providerInfo *v1alpha1.KubernetesClusterSpec_ProviderInfo,
+	kubeClusterLabels map[string]string,
+	policyRules []*v1alpha1.PolicyRule,
```

Comment:
given the number of options we're growing to here, i think it makes sense to encapsulate these in a struct. wdyt?